libr/bin/p/bin_pe.c

```diff
@@ -613,6 +613,15 @@ static RBuffer* create(RBin* bin, const ut8 *code, int codelen, const ut8 *data,
 	return buf;
 }
 
+static char *signature (RBinFile *arch) {
```

Comment:
the definition says that it has to return a `char*`, not a `const char*`; i wanted to change that, but i didn't to avoid problems while merging.